sdk-core/src/main/java/com/reach5/identity/sdk/core/models/requests/webAuthn/AuthenticationPublicKeyCredential.kt

```diff
@@ -0,0 +1,44 @@
+package com.reach5.identity.sdk.core.models.requests.webAuthn
+
+import android.os.Parcelable
+import com.google.android.gms.fido.fido2.api.common.AuthenticatorAssertionResponse
+import com.google.gson.annotations.SerializedName
+import com.reach5.identity.sdk.core.utils.WebAuthn.encodeToBase64
+import com.reach5.identity.sdk.core.utils.WebAuthn.publicKeyCredentialType
+import kotlinx.android.parcel.Parcelize
+
+object WebAuthnAuthentication{
+    fun createAuthenticationPublicKeyCredential(authenticatorAssertionResponse: AuthenticatorAssertionResponse): AuthenticationPublicKeyCredential {
+        return AuthenticationPublicKeyCredential(
+            id = encodeToBase64(authenticatorAssertionResponse.keyHandle),
+            rawId = encodeToBase64(authenticatorAssertionResponse.keyHandle),
```

Comment:
id and rawId are equals? It's intended?

Reply:
`raw_id` is the binary form of the `id`, but we need to keep both.

Do you confirm @rchapel ?

Reply:
Yes they are the same. We are keeping both to keep messages as close as possible from the specification.
Actually `rawId` is a byte buffer and `id` the Base64 encoding from this byte buffer.

Reply:
So @rchapel is it ok if I encode in Base64 the `rawId` too?

Reply:
Yes if it is needed. But be careful to not encode twice the id in base 64 (if it is already base64 encoded).

Reply:
No worries, they are not. If I don't encode them I'll have the error `bad base64 encoding` from the server.